mocha_test/asyncify.js

```diff
@@ -74,6 +74,13 @@ describe('asyncify', function(){
             'rsvp'
         ];
 
+        // Both Bluebird and native promises emit these events. We handle it because Bluebird
+        // will report these rejections to stderr if we don't, which is a great feature for
+        // normal cases, but not here, since we expect unhandled rejections:
+        process.on('unhandledRejection', function () {
+            // Ignore.
+        });
```

Comment:
Just realized, can you remove this after the test completes? `process.off` or only bind it with `process.once`

Please make a followup pr